src/test/java/io/appium/java_client/pagefactory_tests/widget/tests/combined/CombinedAppTest.java

```diff
@@ -31,26 +29,26 @@ public class CombinedAppTest extends WidgetTest {
      *
      * @return test parameters
      */
-    @Parameterized.Parameters
-    public static Collection<Object[]> data() {
-        return asList(
-                dataArray(new CombinedApp(), new AbstractStubWebDriver.StubAndroidDriver(), DefaultAndroidWidget.class),
-                dataArray(new CombinedApp(), new AbstractStubWebDriver.StubIOSXCUITDriver(),
+
+    public static Stream<Arguments> data() {
```

Comment:
shouldn't this method be referenced in tests via `@MethodSource` annotation

Reply:
Added `@MethodSource` annotation in parent class